src/components/GraphFilter/GraphFilterToolbar.tsx

```diff
@@ -70,7 +61,7 @@ export default class GraphFilterToolbar extends React.PureComponent<GraphFilterT
         // @ts-ignore
         GraphDataActions.fetchGraphData(
           store.getState().namespaces.activeNamespace,
-          this.props.graphDuration,
+          store.getState().userSettings.duration,
```

Comment:
Now that this component has more usages of the `store` we should consider wrapping it in a container to follow the redux patterns, rather than using the store directly.